### PR TITLE
Feature: App Layout

### PR DESCRIPTION
**Pull request by dbrosio3**
## Links:

* [Trello Card](https://trello.com/c/iRUtgL5Z/4-1-app-layout)
* [Mockups](https://www.figma.com/file/79uiALRVOHTMEHyqShdHnA/Black-Market)

## What & Why:

Grabación de pantalla 2022-05-26 a la(s) 16 54 27

Grabación de pantalla 2022-05-26 a la(s) 16 57 42

## Risks, Mitigation & Rollback:

- [] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

**Review by dariokozicki — APPROVED** [review]

File: src/components/Menu/AccountMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          </MenuButton>
+          <AccountMenuList
+            items={[
+              { label: 'Recent Searches' },
+              { label: 'Direct Messages' },
+              { label: 'Purchases' },
+              { label: 'Security' },
+              { label: 'Configuration' },
+              { label: 'Help' },
+              { label: 'Log Out', action: logout },
```
Comment (elizabethlofredo) [inline src/components/Menu/AccountMenu.tsx]: I think those should be on the translation strings

File: src/components/Menu/AccountMenuList.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  label: string;
+  action?: () => void;
+};
+
+export const AccountMenuList: React.FC<{ items: Array<Item> }> = ({ items }) => (
+  <StyledMenuList>
+    {items.map(item => (
+      <StyledMenuItem key={item.label} onClick={item.action}>
+        {item.label}
+      </StyledMenuItem>
```
Comment (elizabethlofredo) [inline src/components/Menu/AccountMenuList.tsx]: small suggestion 
```suggestion
    {items.map(({label, action}) => (
      <StyledMenuItem key={label} onClick={action}>
        {label}
      </StyledMenuItem>
```

File: src/components/Search/SearchBar.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import { Input, InputGroup, InputRightElement } from '@chakra-ui/react';
+
+export const SearchBar = () => (
+  <InputGroup
+    bg="white"
+    w="calc(100% - 382px)"
+    display={{ base: 'none', md: 'block' }}
+    borderRadius="lg"
+  >
+    <Input placeholder="Search for products" h="40px" />
```
Comment (elizabethlofredo) [inline src/components/Search/SearchBar.tsx]: should this be on translation strings?

File: src/components/Widget/NavBarWidget.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+export const NavBarWidget = chakra(WidgetButton, {
+  baseStyle: {
+    height: '39px',
+    width: '168px',
+    minW: 'unset',
+    flexShrink: 0,
+    border: '0.5px solid white',
+    borderTopRadius: 'lg',
+    padding: 0,
+    color: 'white',
```
Comment (elizabethlofredo) [inline src/components/Widget/NavBarWidget.tsx]: WDYT of having this white color on `src/styles/theme/index.ts`?

**Review by elizabethlofredo — APPROVED** [review]
Nice job! I left some small suggestions/comments

File: src/components/Widget/NavBarWidget.tsx (hunk 4 above)
Comment (dbrosio3) [reply src/components/Widget/NavBarWidget.tsx]: Makes sense, thanks!

**Pull request merged**